src/lib/conversions.ts

```diff
@@ -4,7 +4,7 @@ import { EnvVarObject } from './types';
 
 export function envArrayToObject(env: string[]): EnvVarObject {
 	const toPair = (keyVal: string) => {
-		const m = keyVal.match(/^([^=]+)=\s*(.*)\s*$/);
+		const m = keyVal.match(/^([^=]+)=((?:.|\n)*)$/);
```

Comment:
You can use the `s` flag instead:
```suggestion
		const m = keyVal.match(/^([^=]+)=(.*)$/s);
```
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions#Advanced_searching_with_flags_2

Reply:
@Page- we can't use this, as it's not supported by the node version on `i386-nlp`.

Reply:
Ah that sucks, in that case it's probably better to use
```suggestion
		const m = keyVal.match(/^([^=]+)=([\s\S]*)$/);
```
as an equivalent that will match everything (your current version will not match `\r`)